evap/staff/templates/staff_evaluation_textanswers_section.html

```diff
@@ -33,9 +33,39 @@
                                 </td>
                                 <td>
                                     <div id="{{ answer.id }}-buttons" class="btn-group">
-                                        <button type="button" id="{{ answer.id }}-publish" onclick="press('{{ answer.id }}', 'publish');" class="btn btn-sm btn-light{% if answer.is_published %} active{% endif %}">{% trans 'yes' %}</button>
-                                        {% if contributor %}<button type="button" id="{{ answer.id }}-make_private" onclick="press('{{ answer.id }}', 'make_private');" class="btn btn-sm btn-light{% if answer.is_private %} active{% endif %}">{% trans 'private' %}</button>{% endif %}
-                                        <button type="button" id="{{ answer.id }}-hide" onclick="press('{{ answer.id }}', 'hide');" class="btn btn-sm btn-light{% if answer.is_hidden %} active{% endif %}">{% trans 'no' %}</button>
+                                        <form class="full-update-textanswer-form" method="POST" action="{% url 'staff:evaluation_textanswers_update_publish' %}">
+                                            {% csrf_token %}
+
+                                            <input type="hidden" name="id" value="{{ answer.id }}" />
+                                            <input type="hidden" name="evaluation_id" value="{{ evaluation_id }}" />
+
+                                            <button
+                                                type="submit"
+                                                name="action"
+                                                value="publish"
+                                                class="btn btn-sm btn-light{% if answer.is_published %} active{% endif %}"
+                                            >
```

Comment:
much lines, many wow :)
do we really want to have everything on a new line?

Reply:
I don't know, I think this one is more readable, but we can also use the one-line formatting. Prettier [agrees with me](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeARgVxjaACGATwAc4BeAHRAGdN0BbASxirygEN7yr2wZHorAG7sANpm4hidUY2oALVmFHtq1SiHQwoeLVAC01eru37ZAc3kxgAUjyMAZnnZRqAdzgAnAHRyA+tLosgpwACZ4NgC+znyMQnC2eAihjhGRVAB8FFAURHBwLFCoAPRYONBZUCAANCAQxPzQ1Mig7J6eEG4ACm0IzShibuyEzbXonrwA1gUAypxwADKMUHDIDmLUcGMTYNMwM8S8y+bIMJ4StXD06GGhYQsu5pjs5nAAYhCe9Ow4x8gg7GwEBqICs9FEAHV5Mw4NRDmA4DM+sw4sxCP8wGoQctNp4YF0JuZvmsNlsQAAragADxmx1EcAAipgIPASaJNrVDp5cf8waIQcRPMsYBDGKEYPJkABGABMAAZOR1NhCJsR-oLYV54iCAI7M+AE+r9AHUfQrMJhEGeOB6xjWgkvYlIdbssmbJinc5uumM-WrZ2k2owdjoUXiyVIGVBiaMWRQcwAYQg9CdIFhAFYQZhNgAVEP9F0ckBCCQASSgd1gMzAQoaAEEKzMiPS2ZtIpEgA) though :D